lib/gems/pending/appliance_console/external_httpd_configuration.rb

```diff
@@ -106,6 +106,9 @@ def configure_sssd_domain(config, domain)
           pattern = "[domain/#{Regexp.escape(domain)}].*(\n)"
           config[/#{pattern}/, 1] = "\nldap_user_extra_attrs = #{ldap_user_extra_attrs}\n"
         end
+
+        pattern = "ldap_user_extra_attrs.*(\n)"
+        config[/#{pattern}/, 1] = "\nentry_cache_timeout = 600\n"
```

Comment:
just for correctness, I wonder if we shouldn't match that pattern in the domain section specifically.

```
pattern = "[domain/#{Regexp.escape(domain)}](\n.*)+ldap_user_extra_attrs.*(\n)"
config[/#{pattern}/,2] = "\nentry_cache_timeout = 600\n" 
```